src/test/unit/utils.test.ts

```diff
@@ -1,5 +1,5 @@
 import { expect, assert } from 'chai'
-import { sleep, getEventFromTx } from '../../utils/util.js' // replace './yourModule' with the actual path of your module
+import { sleep, getEventFromTx, isValidUrl } from '../../utils/util.js' // replace './yourModule' with the actual path of your module
```

Comment:
Can you remove the pointless comment:
 
```suggestion
import { sleep, getEventFromTx, isValidUrl } from '../../utils/util.js'
```

Reply:
strange.. that's already on develop branch, no idea where it comes from, seems automatic stuff